src/external/dftk_to_wannier90.jl

```diff
@@ -0,0 +1,605 @@
+    push!(angular_parts, [3,4] => (θ,φ) -> (√105/(4*√π))   * (sin(θ)^2)*cos(θ)*cos(2*φ) )                #fz(x2-y2)
+    push!(angular_parts, [3,5] => (θ,φ) -> (√105/(4*√π))   * (sin(θ)^2)*cos(θ)*sin(2*φ) )                #fxyz
+    push!(angular_parts, [3,6] => (θ,φ) -> (√35/(4*√(2π))) * (sin(θ)^3)*(cos(φ)^2 - 3*sin(φ)^2)*cos(φ) ) #fx(x2-3y2)
+    push!(angular_parts, [3,7] => (θ,φ) -> (√35/(4*√(2π))) * (sin(θ)^3)*(3*cos(φ)^2 - sin(φ)^2)*sin(φ) ) #fy(3x2-y2)
+        
+    push!(angular_parts, [-1,1] => (θ,φ) -> (1/√2)*s + (1/√2)*px ) #sp-1 
+    push!(angular_parts, [-1,2] => (θ,φ) -> (1/√2)*s - (1/√2)*px ) #sp-2
+    
+    push!(angular_parts, [-2,1] => (θ,φ) -> (1/√3)*s - (1/√6)*px + (1/√2)*py) #sp2-1
+    push!(angular_parts, [-2,2] => (θ,φ) -> (1/√3)*s - (1/√6)*px - (1/√2)*py) #sp2-2
+    push!(angular_parts, [-2,3] => (θ,φ) -> (1/√3)*s + (2/√6)*px )            #sp2-3
+    
+    push!(angular_parts, [-3,1] => (θ,φ) -> 0.5*(s(θ,φ)+px(θ,φ)+py(θ,φ)+pz(θ,φ)) )  #sp3-1
+    push!(angular_parts, [-3,2] => (θ,φ) -> 0.5*(s(θ,φ)+px(θ,φ)-py(θ,φ)-pz(θ,φ)) )  #sp3-2
+    push!(angular_parts, [-3,3] => (θ,φ) -> 0.5*(s(θ,φ)-px(θ,φ)+py(θ,φ)-pz(θ,φ)) )  #sp3-3
+    push!(angular_parts, [-3,4] => (θ,φ) -> 0.5*(s(θ,φ)-px(θ,φ)-py(θ,φ)+pz(θ,φ)) )  #sp3-4
+
+    push!(angular_parts, [-4,1] => (θ,φ) ->  (1/√3)*s - (1/√6)*px + (1/√2)*py)  #sp3d-1
+    push!(angular_parts, [-4,2] => (θ,φ) ->  (1/√3)*s - (1/√6)*px - (1/√2)*py)  #sp3d-2
+    push!(angular_parts, [-4,3] => (θ,φ) ->  (1/√3)*s + (2/√6)*px)              #sp3d-3
+    push!(angular_parts, [-4,4] => (θ,φ) ->  (1/√2)*pz + (1/√2)*dz2 )           #sp3d-4
+    push!(angular_parts, [-4,5] => (θ,φ) -> -(1/√2)*pz + (1/√2)*dz2 )           #sp3d-5
+
+
+    push!(angular_parts, [-5,1] => (θ,φ) -> (1/√6)*s - (1/√2)*px - (1/√12)*dz2 +  (1/2)*dx2_y2) #sp3d2-1
+    push!(angular_parts, [-5,2] => (θ,φ) -> (1/√6)*s + (1/√2)*px - (1/√12)*dz2 +  (1/2)*dx2_y2) #sp3d2-2
+    push!(angular_parts, [-5,3] => (θ,φ) -> (1/√6)*s - (1/√2)*py - (1/√12)*dz2 -  (1/2)*dx2_y2) #sp3d2-3
+    push!(angular_parts, [-5,4] => (θ,φ) -> (1/√6)*s + (1/√2)*py - (1/√12)*dz2 -  (1/2)*dx2_y2) #sp3d2-4
+    push!(angular_parts, [-5,5] => (θ,φ) -> (1/√6)*s - (1/√2)*pz + (1/√3)*dz2)                  #sp3d2-5
+    push!(angular_parts, [-5,6] => (θ,φ) -> (1/√6)*s + (1/√2)*pz + (1/√3)*dz2)                  #sp3d2-6
+    
+ 
+
+    radial_parts,angular_parts
+end
+
+
+
+""" 
+    Takes one line of the projections table given by read_nnkp_file and  dictionaries linking quantum numbers
+    and radial or angular parts of hydrogen AOs, and produce the analytic expression of the guess.
+    Remember that : proj = [ [center], [quantum numbers], [z_axis], [x_axis], α ]
+"""
+function guess_win(proj)
+    center = proj[1]
+    l,mr,r = proj[2]
+    α = proj[5]
+
+    radial_parts,angular_parts = dict_AOs(α)
+    #TODO take into account non-canonical basis
+    @assert(proj[3] == [0.00,0.00,1.00]) #For now we limit ourselves to the canonical basis
```

Comment:
For comparisons you can just do `[0, 0, 1]`: `[0.0, 0.0, 1.0] == [0, 0, 1]` is true